packages/lab/src/__tests__/__e2e__/badge/Badge.cy.tsx

```diff
@@ -2,53 +2,37 @@ import { Badge } from "@salt-ds/lab";
 
 describe("GIVEN a Badge", () => {
   it("THEN can render a Badge with badge number larger than max", () => {
-    cy.mount(<Badge badgeContent={100} max={99} />);
+    cy.mount(<Badge value={100} max={99} />);
     cy.findByText("99+").should("exist");
   });
 
   it("THEN can render a Badge with badge number equal to max", () => {
-    cy.mount(<Badge badgeContent={99} max={99} />);
+    cy.mount(<Badge value={99} max={99} />);
     cy.findByText("99").should("exist");
   });
 
   it("THEN can render a Badge with badge number smaller than max", () => {
-    cy.mount(<Badge badgeContent={98} max={99} />);
+    cy.mount(<Badge value={98} max={99} />);
     cy.findByText("98").should("exist");
   });
 
-  it("THEN can render a Badge with default icon", () => {
-    cy.mount(<Badge badgeContent={98} />);
-    cy.findByRole("img", { name: "message" }).should("exist");
+  it("THEN cannot render a Badge with value greater than 999 ", () => {
+    cy.mount(<Badge value={1000} />);
+    cy.findByText("999+").should("exist");
   });
 
-  it("THEN can render with a custom text child", () => {
-    cy.mount(<Badge badgeContent={1}>Lorem Ipsum</Badge>);
-    cy.findByText("Lorem Ipsum").should("exist");
+  it("THEN can render a string", () => {
+    cy.mount(<Badge value={"lots"} />);
+    cy.findByText("lots").should("exist");
   });
 
-  it("SHOULD reference the child and badge content with aria-labelledby", () => {
-    cy.mount(
-      <Badge badgeContent="BADGE_CONTENT">
-        <div data-testid="badgeChild" />
-      </Badge>
-    );
-
-    cy.findByTestId("badgeChild").invoke("attr", "id").as("childId");
-    cy.findByText("BADGE_CONTENT").invoke("attr", "id").as("contentId");
-
-    cy.get("@childId").then((childId) => {
-      cy.get("@contentId").then((contentId) => {
-        cy.findByRole("img").should(
-          "have.attr",
-          "aria-labelledby",
-          `${childId} ${contentId}`
-        );
-      });
-    });
+  it("THEN can render with a custom text child", () => {
+    cy.mount(<Badge value={1}>Lorem Ipsum</Badge>);
+    cy.findByText("Lorem Ipsum").should("exist");
   });
 
-  it("SHOULD have no a11y violations on load", () => {
-    cy.mount(<Badge />);
-    cy.checkAxeComponent();
+  it("THEN can render with a custom text child", () => {
+    cy.mount(<Badge value={1}>Lorem Ipsum</Badge>);
+    cy.findByText("Lorem Ipsum").should("exist");
   });
```

Comment:
Try this locally first if it works
```suggestion
  it("THEN it truncates a string longer than 4 characters", () => {
    cy.mount(<Badge value={1}>Lorem Ipsum</Badge>);
    cy.findByText("Lorem Ipsum").should("exist").should("not.be.visible");
    cy.findByText("Lore...").should("exist").should("be.visible").should("have.attr", "aria-hidden");
  });
```

Reply:
Ok, good enough. I was hoping the `should("not.be.visible")` would work